pom.xml

```diff
@@ -24,10 +24,10 @@
   
   <dependencies>
     <dependency>
-      <groupId>com.cdancy</groupId>
-      <artifactId>jenkins-rest</artifactId>
-      <version>0.0.10</version>
-    </dependency>
+			<groupId>com.offbytwo.jenkins</groupId>
+			<artifactId>jenkins-client</artifactId>
+			<version>0.3.7</version>
+		</dependency>
```

Comment:
please indent correctly, and move this to the bot/pom.xml instead of this root pom.xml (yet still remove jenkins-rest from the root pom.xml)